gridcoinresearch.pro

```diff
@@ -19,6 +19,7 @@ greaterThan(QT_MAJOR_VERSION, 4) {
 win32:CONFIG += qaxcontainer
 CONFIG += exceptions
 
+QT += charts concurrent
```

Comment:
I was thinking in order to get the ball rolling, maybe we can do a conditional build so we only show charts when building for Qt >=5.7. With older versions you'd simply get a sorted list of the results:

- "Yes, go ahead": 57%
- "No idea": 30%
- "No, NO!": 13%